### ISSUE TITLE
Add automatic creation of credentials.pem

### ISSUE THREAD

**Issue by basilveerman** [pacificclimate/esgf_download#1]
Before downloads can be fetched, credentials must be generated using:

`myproxyclient logon -s pcmdi9.llnl.gov -b -C ~/.esg/certificates -o ~/.esg/credentials.pem -l USERNAME`

This isn't really necessary and can be incorporated into the esgf_fetch_downloads.py

**Comment by sterlingbaldwin** [comment]
I believe this functionality is built into the LogonManager from pyesgf.logon, just logon using the boostrap=True argument.

    from pyesgf.logon import LogonManager
    lm = LogonManager()
    lm.logon_with_openid('username', 'password', bootstrap=True)

**Comment by basilveerman** [comment]
Exactly, thanks. We just don't currently [use that option](https://github.com/pacificclimate/esgf_download/blob/ddd21bd4f12e23a2c691da6f067d89d8366c7e5c/esgf_download/__init__.py#L464). I'd also want to pass it as a command line option rather than an unchangeable default.
